contracts/ChannelManager.sol

```diff
@@ -218,16 +216,19 @@ contract ChannelManager {
     {
         Channel storage channel = channels[channelId];
 
-        // sanity checks
+        // sanity checks;
+        uint256[] bals;
+        bals.push(balanceA);
+        bals.push(balanceB);
```

Comment:
I'm not a solidity expert but it might make the code less readable. What if you'd do a `struct BalancePair { uint256 a, uint256 b }` and pass it-  would it change the amount of local variables and stack size?

Reply:
Yes that is great. But did you try this one out first?

Reply:
@mpapierski 
Struct is used at global level, and fixing stack deep is for array